src/main/java/com/stripe/param/checkout/SessionCreateParams.java

```diff
@@ -0,0 +1,1149 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param.checkout;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+public class SessionCreateParams extends ApiRequestParams {
+  /**
+   * Specify whether Checkout should collect the customer's billing address. If set to `required`,
+   * Checkout will always collect the customer's billing address. If left blank or set to `auto`
+   * Checkout will only collect the billing address when necessary.
+   */
+  @SerializedName("billing_address_collection")
+  BillingAddressCollection billingAddressCollection;
+
+  /**
+   * The URL the customer will be directed to if they decide to cancel payment and return to your
+   * website.
+   */
+  @SerializedName("cancel_url")
+  String cancelUrl;
+
+  /**
+   * A unique string to reference the Checkout Session. This can be a customer ID, a cart ID, or
+   * similar, and can be used to reconcile the session with your internal systems.
+   */
+  @SerializedName("client_reference_id")
+  String clientReferenceId;
+
+  /**
+   * ID of an existing customer paying for this session, if one exists. May only be used with
+   * line_items. Usage with subscription_data is not yet available. If blank, Checkout will create a
+   * new customer object based on information provided during the session. The email stored on the
+   * customer will be used to prefill the email field on the Checkout page. If the customer changes
+   * their email on the Checkout page, the Customer object will be updated with the new email.
+   */
+  @SerializedName("customer")
+  String customer;
+
+  /**
+   * If provided, this value will be used when the Customer object is created. If not provided,
+   * customers will be asked to enter their email address. Use this parameter to prefill customer
+   * data if you already have an email on file. To access information about the customer once a
+   * session is complete, use the `customer` field.
+   */
+  @SerializedName("customer_email")
+  String customerEmail;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * A list of items the customer is purchasing. Use this parameter for one-time payments. To create
+   * subscriptions, use `subscription_data.items`.
+   */
+  @SerializedName("line_items")
+  List<LineItem> lineItems;
+
+  /**
+   * The IETF language tag of the locale Checkout is displayed in. If blank or `auto`, the browser's
+   * locale is used. Supported values are `auto`, `da`, `de`, `en`, `es`, `fi`, `fr`, `it`, `ja`,
+   * `nb`, `nl`, `pl`, `pt`, `sv`, or `zh`.
+   */
+  @SerializedName("locale")
+  Locale locale;
```

Comment:
@ob-stripe Do you think we should change this to "SupportedLocale" or "LanguageLocale", because name might conflate with `java.util.Locale`?

Reply:
Thank you ob for the note. 
Also with https://errorprone.info/bugpattern/JavaLangClash that `util` pacakge is okay.